src/BucketInterface.php

```diff
@@ -21,4 +21,11 @@ public function enabled(BehaviorInterface $behavior);
      * @return int
      */
     public function getIndex();
+
+    /**
+     * Sets the callback
+     *
+     * @param callable $callback
+     */
+    public function setMissingSlugCallback(callable $callback);
```

Comment:
@slash28cu Adding this method to the interface makes the change backwards incompatible.  In lieu of bumping the major version, maybe we can just accept the optional callback in the constructor for now.